crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -174,7 +174,7 @@ async fn protocol_fee_calculated_on_the_solver_side() {
                 buy: 40.ether().into_wei(),
             },
         },
-        expected_score: 20.ether().into_wei(),
+        expected_score: 35.ether().into_wei(),
```

Comment:
Now even if the `Solver` calculates the protocol fee, we still need to count it for the score, as per:

```rust
    /// CIP38 score defined as surplus + protocol fee
    ///
    /// Denominated in NATIVE token
    fn score(&self, prices: &auction::Prices) -> Result<eth::Ether, Error> {
        tracing::debug!("Scoring trade {:?}", self);
        Ok(self.native_surplus(prices)? + self.native_protocol_fee(prices)?)
    }
```

Reply:
Not too obvious from the numbers in the test that we get capped by volume, but the math seems correct.

Reply:
Would it be possible to add comments to the test case explaining the expected values?